keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -252,222 +252,461 @@ non-goals of this KEP.
 type DeviceMixinRef struct {
-	// Name refers to the name of a device mixin in the pool.
-	//
-	// +required
-	Name string `json:"name"`
+  // Name refers to the name of a device mixin in the pool.
+  //
+  // +required
+  Name string
+}
+
+// CapacityPoolMixinRef defines a reference from a capacity pool
+// to a capacity pool mixin.
+type CapacityPoolMixinRef struct {
+  // Name is the name of a CapacityPoolMixin.
+  //
+  // +required
+  Name string
 }
 
-// DeviceRef defines a reference to a device.
-type DeviceRef struct {
-	// Name refers to the name of a device in the pool.
-	//
-	// +required
-	Name string `json:"name"`
+type DeviceCapacityConsumptionMixinRef struct {
+  // Name is the name of a DeviceCapacityConsumptionMixin.
+  //
+  // +required
+  Name string
+}
+
+// DeviceCapacityConsumptionRef defines a set of capacities that
+// a device will consume from a capacity pool.
+type DeviceCapacityConsumptionRef struct {
+  // CapacityPool defines the capacity pool from which the
+  // capacities defined (either directly or through a mixin)
+  // will be consumed from.
+  //
+  // +required
+  CapacityPool string
+  
+  // Mixins defines a list of references to DeviceCapacityConsumptionMixins.
+  // The capacities listed in these will be included in among the
+  // capacities that will be consumed by the device.
+  //
+  // Capacities listed directly will override any capacities coming
+  // from mixins.
+  //
+  // +optional
+  Mixins []DeviceCapacityConsumptionMixinRef
+
+  // CapacityConsumption defines the capacity that will be consumed by
+  // the device.
+  //
+  // Capacity listed here will override any capacities that
+  // are also defined in any of the referenced mixins.
+  //
+  // +optional
+  CapacityConsumption DeviceCapacity
 }
 ```
 
 As mentioned previously, the main features being added here are (1) the ability
-to include a set of mixins in a device definition, and (2) the ability to
-express that capacity from one device gets consumed by another device if/when
-the scheduler decides to allocate it.
+to include a set of mixins in a device or capacity pool definition, and (2) the
+ability to express that multiple devices draw from the same pool of capacity, so
+allocation of one device might make other devices unallocatable.
 
 To simplify the conversation, we discuss each new feature separately, starting
-with "mixins" and the new `Includes` field, which allows a set of mixins to
-extend a device with common attributes and capacities.
+with "mixins" for both capacity pools and devices, which allows a set of mixins to
+extend the properties defined explicitly.
 
-### Extending a device with as set of mixins
+### Extending device and capacity pool with a set of mixins
 
 A simple example the defines a set of mixins and includes them in the
 definition of 4 NVIDIA A100 GPUs can be seen below:
 
 ```yaml
+capacityPoolMixins:
+- name: gpu-pool-mixin
+  capacity:
+    memory: 40Gi
```

Comment:
I believe this should be (all throughout):
```
  capacity:
    memory:
      value: 40Gi
```

Reply:
I've cleaned up the updated examples and will make sure do it this way for the rest of the examples.